src/main/asciidoc/_chapters/configuration.adoc

```diff
@@ -636,7 +636,9 @@ read by HBase daemons on startup.
 Changes here will require a cluster restart for HBase to notice the change.
 
 [[log4j]]
-=== _log4j.properties_
+=== _log4j2.xml_
+
+Since version 3.0.0, HBase has upgraded to Log4j2, so the configuration file name and format has changed. Read more in link:https://logging.apache.org/log4j/2.x/index.html[Apache Log4j2].
 
```

Comment:
Was wondering if this onl y place we mention log4j?

Maybe a few more sentences ? What is the basic diff

Reply:
This is a good suggestion, I will check other places mentioned to log4j and make some additional instructions for this.

Reply:
Should we revert this commit so that it can include any additional references?